lua/gitlinker/git.lua

```diff
@@ -144,7 +148,7 @@ local function parse_repo_path(stripped_uri, host, port, errs)
   end
 
   -- add parsed host to path capture
-  path_capture = host .. path_capture
+  path_capture = hyphens_escaped(host) .. path_capture
```

Comment:
same philosophy here

```suggestion
  path_capture = chars .. "+" .. path_capture
```